src/main/java/uk/gov/hmcts/reform/blobrouter/data/envelopes/EnvelopeRepository.java

```diff
@@ -159,4 +159,14 @@ public Integer getEnvelopesCount(Set<String> containers, Instant fromDateTime, I
             Integer.class
         );
     }
+
+    public List<Envelope> getEnvelopes(Instant fromDateTime, Instant toDateTime) {
+        return jdbcTemplate.query(
+            "SELECT * FROM envelopes WHERE file_created_at BETWEEN :fromDateTime AND :toDateTime",
+            new MapSqlParameterSource()
+                .addValue("fromDateTime", Timestamp.from(fromDateTime))
+                .addValue("toDateTime", Timestamp.from(toDateTime)),
+            this.mapper
+        );
+    }
```

Comment:
if we actually want just envelopes from given DAY then there's no need for those 2 params, just pass `LocalDate`

Reply:
the updated method is using just `LocalDate`